feat: add story telling workshop (#56978)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-story-teller-app/671fae2d8e402a405a930e44.md

---
id: 671fae2d8e402a405a930e44
title: Step 11
challengeType: 0
dashedName: step-11
---

# --description--

> Add the `displayStory` function to the `click` event listener of the `scaryStoryBtn`, and see the output in the console.

Reviewer: ```suggestion
Add an event listener for the `click` event to `scaryStoryBtn` and pass it the `displayStory` function. Then, click your button and see the output in the console.
```

# --hints--

> You should add the `displayStory` function to the `click` event listener of the `scaryStoryBtn` and see the output `"You clicked the button"` in the console when you click the `scaryStoryBtn`.

Reviewer: ```suggestion
You should add an event listener for the `click` event to `scaryStoryBtn` passing it the `displayStory` function.
```
